### PR TITLE
Create go.k8s.io/stuck-prs

### PR DESCRIPTION
**Pull request by fejta**
/assign @ixdy @rmmh @cjwagner

### CODE REVIEW

File: k8s.io/configmap-nginx.yaml [hunk 1]
```diff
@@ -200,10 +200,11 @@ data:
...
           rewrite ^/oncall$          https://storage.googleapis.com/kubernetes-jenkins/oncall.html redirect;
           rewrite ^/partner-request$ https://docs.google.com/forms/d/e/1FAIpQLSdN1KtSKX2VAOPGABFlShkSd6CajQynoL4QCVtY0dj76MNDKg/viewform redirect;
-          rewrite ^/start$           http://kubernetes.io/docs/getting-started-guides/ redirect;
+          rewrite ^/start$           https://kubernetes.io/docs/getting-started-guides/ redirect;
+          rewrite ^/stuck-prs$       https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Aopen%20label%3Algtm%20label%3Aapproved%20-label%3Ado-not-merge%20-label%3Aneeds-rebase%20sort%3Aupdated-asc%20-status%3Asuccess redirect;
           rewrite ^/test-history$    https://storage.googleapis.com/kubernetes-test-history/static/index.html redirect;
           rewrite ^/triage$          https://storage.googleapis.com/k8s-gubernator/triage/index.html redirect;
-          rewrite ^/test-health$     http://velodrome.k8s.io/dashboard/db/bigquery-metrics redirect;
+          rewrite ^/test-health$     https://velodrome.k8s.io/dashboard/db/bigquery-metrics redirect;
```
Comment (cjwagner) [inline k8s.io/configmap-nginx.yaml]: I don't think Velodrome uses HTTPS. That link doesn't seem to work.

Comment (fejta) [inline k8s.io/configmap-nginx.yaml]: Done

Comment (ixdy) [inline k8s.io/configmap-nginx.yaml]: did you mean to revert https for start?

**Comment by fejta** [comment]
@ixdy will you review/merge this?

**Comment by ixdy** [comment]
@fejta yes, I left a comment above

File: k8s.io/configmap-nginx.yaml (hunk 1 above)
Comment (fejta) [inline k8s.io/configmap-nginx.yaml]: yes

**Comment by ixdy** [comment]
tests pass on canary cluster, LGTM

**Comment by ixdy** [comment]
deployed: https://go.k8s.io/stuck-prs

**Comment by fejta** [comment]
thanks!

**Pull request merged**